buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/TestClustersPlugin.java

```diff
@@ -67,14 +72,25 @@ public void apply(Project project) {
 
         // There's a single Gradle instance for multi project builds, this means that some configuration needs to be
         // done only once even if the plugin is applied multiple times as a part of multi project build
-        ExtraPropertiesExtension rootProperties = project.getRootProject().getExtensions().getExtraProperties();
-        if (rootProperties.has(PROPERTY_TESTCLUSTERS_RUN_ONCE) == false) {
-            rootProperties.set(PROPERTY_TESTCLUSTERS_RUN_ONCE, true);
+        if (rootProject.getConfigurations().findByName(HELPER_CONFIGURATION_NAME) == null) {
+            Configuration helperConfiguration = project.getRootProject().getConfigurations().create(HELPER_CONFIGURATION_NAME);
```

Comment:
What is the advantage of sticking the helper configuration on the root project?

Reply:
I think it'd be worth a comment explaining it. I'm sort of used to sticking the configurations that I need on the project that needs them.

Reply:
Most of the build will use the same distros and plugins, by having it there we only resolve the configuration and extract once. I'm also hoping to be able to start up clusters from there on COW if not always possible. I'll add a comment.